nameko_eventlog_dispatcher/eventlog_dispatcher.py

```diff
@@ -28,20 +28,31 @@ class EventLogDispatcher(EventDispatcher):
     ENTRYPOINT_FIRED = 'entrypoint_fired'
     """Event type used by events triggered when entrypoints are fired."""
 
-    EVENT_TYPE = 'log_event'
+    GENERIC_EVENT_TYPE = 'log_event'
```

Comment:
Or maybe we can make this `NAMEKO_EVENT_TYPE` (and change the config key to `nameko_event_type`).